tests/CompositeChat/CompositeChatTest.cpp

```diff
@@ -109,6 +109,7 @@ static void TestParser5(void)
 
 
 
+
```

Comment:
This was a style violation caught by my pre-commit hook during the merge; easier just to fix it.